myhpi/core/templatetags/core_extras.py

```diff
@@ -35,20 +35,6 @@ def insert_footer(page):
     return {"footer_columns": [footer.column_1, footer.column_2, footer.column_3], "page": page}
 
 
-@register.filter(name="tag_external_links")
-def tag_external_links(content):
-    """Takes the content of a website and inserts external link icons after every external link."""
-    external_links = re.finditer('<a[^>]*href="(?!' + settings.SITE_URL + ")[^>]*>[^<]*", content)
```

Comment:
this also checked whether the link is part of our page by comparing to `settings.SITE_URL`. You could also check that, as a manual link (e.g. if I entered "myhpi.de/fachschaftsrat" into a page) would also be labeled as external if I understand your code correctly